sources/packages/api/src/services/institution-location/institution-location.service.ts

```diff
@@ -20,16 +20,16 @@ export class InstitutionLocationService extends RecordDataModelService<Instituti
 
   async getInstitutionLocationById(id: number): Promise<InstitutionLocation> {
     return this.repo
-      .createQueryBuilder("institution_location")
+      .createQueryBuilder("location")
```

Comment:
As discussed, we can reactor during our Typeorm query refactoring.

Reply:
As discussed not refactoring in this ticket.